benchmark/ts/strip-types.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const path = require('path');
+const assert = require('node:assert');
+
+const js = path.resolve(__dirname, '../fixtures/strip-types-benchmark.js');
+const ts = path.resolve(__dirname, '../fixtures/strip-types-benchmark.ts');
+
+const bench = common.createBenchmark(main, {
+  filepath: [ts, js],
+  n: [1e4],
+}, {
+  flags: ['--experimental-strip-types', '--disable-warning=ExperimentalWarning'],
+});
+
+async function main({ n, filepath }) {
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    const { result } = await import(filepath);
+    assert.ok(result);
```

Comment:
```suggestion
```

Reply:
The `assert.ok` should be outside the benchmark, as it's not being benchmarked, the `import` statement is.